install.conf

```diff
@@ -78,4 +78,12 @@ ACCESS_PRIVATE_KEY_PATH="/root/.ssh/id_rsa"
 
 OFFLINE_INSTALL="false"
 
-OFFLINE_PKG_PATH=""
\ No newline at end of file
+OFFLINE_PKG_PATH=""
+
+#######################################################################
+# runtime config
+# if containerd_runtime is false, then use docker
+# else, then use containerd
+#######################################################################
+
+CONTAINERD_RUNTIME="false"
```

Comment:
Better use `CRI=docker` instead of `CONTAINERD_RUNTIME=false`, cause we maybe support more CRIs in the future.